english/discover-new-genres/grime/grime.md

```diff
@@ -0,0 +1,37 @@
+You have decided to learn about grime! Grime is a genre originating from the
+United Kingdom that mainly grew out of similar, older genres of electronic music
+in the UK - primarily UK garage and jungle. Grime music is normally set at a
+tempo of 140 beats per minute and feature dark basslines, futuristic electronic
+noises, and an abundance of square waves. Grime is listened to both as stand-
+alone instrumentals and with an MC, whose role in the genre has morphed from
+something similar to the literal definition of "mic controller" to something
+very similar to a modern hip-hop rapper. Grime is known for being fast,
+agressive, and hard hitting and although many MCs have moved closer and closer
+to a hip-hop sound, grime still maintains a distinct, underground sound.
+
+Examples:
+
+[Skepta-That's Not Me](https://www.youtube.com/watch?v=dyONbqggasY)
+
+[Wiley-Outchea](https://soundcloud.com/richard-kylea-cowie/wiley-outchea)
+
+[Spooky-Top 10 Dubplate Mix](https://soundcloud.com/factmag/dj-spooky-top-10-dubplate-mix)
+
+[Dark0-Black Rose](https://www.youtube.com/watch?v=-irjIr2jBlM)
+
+[DJ Slimzee, Wiley and Dizzee Rascal-Sidewinder 2003 Set](https://www.youtube.com/watch?v=2IzqD-z8trY)
+
+[Dizzee Rascal-I Luv U](https://www.youtube.com/watch?v=YH0KWX2a8zY)
+
+[Jammz-The World](https://soundcloud.com/iamgrime/jammz-iag001-b1-the-world)
+
+[Danny Weed-Salt Beef](https://www.youtube.com/watch?v=LvBIazVjdn8)
+
+
+Links:
+
+[Wikipedia article](https://en.wikipedia.org/wiki/Grime_music)
+
+[Short documentary on grime history](http://www.factmag.com/2014/11/06/watch-a-full-length-documentary-on-the-history-of-grime-featuring-dizzee-rascal-slimzee-kano-and-more/)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.